specification/desktopvirtualization/resource-manager/readme.java.md

```diff
@@ -1,118 +1,7 @@
 ## Java
 
 These settings apply only when `--java` is specified on the command line.
-Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
 
-``` yaml $(java)
-azure-arm: true
-fluent: true
-namespace: com.microsoft.azure.management.desktopvirtualization
-license-header: MICROSOFT_MIT_NO_CODEGEN
-payload-flattening-threshold: 1
-output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-desktopvirtualization
+```yaml $(java)
+generate-async-methods: true
 ```
-
-### Java multi-api

```

Comment:
clean-up for track2
- Java SDK no longer support multi-api
- other options already configured as default